src/util/getStripeClient.spec.ts

```diff
@@ -5,10 +5,22 @@ import { getStripeClient } from './getStripeClient';
 describe('getStripeClient', () => {
   const apiKey = 'test';
 
-  describe('when `apiKey` is provided', () => {
-    it('should return the stripe client', () => {
-      const stripeClient = getStripeClient({ apiKey });
-      expect(stripeClient).toBeInstanceOf(Stripe);
+  it('should return the stripe client', () => {
+    const stripeClient = getStripeClient({ apiKey });
+    expect(stripeClient).toBeInstanceOf(Stripe);
+  });
+
+  it('should return the stripe client with customize', () => {
+    const stripeClient = getStripeClient({
+      apiKey,
+      appInfo: {
```

Comment:
https://github.com/stripe/stripe-node/wiki/Migration-guide-for-v8#add-appinfo-to-config-object